docs/content/packages/styled-system.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Styled System
+status: Experimental 
+source: 'https://github.com/primer/octicons/tree/v2/lib/octicons_styled'
+---
+
+[![npm version](https://img.shields.io/npm/v/@primer/styled-octicons/canary.svg)](https://www.npmjs.org/package/@primer/styled-octicons)
+
+The `@primer/styled-octicons` package wraps icon components from [`@primer/octicons-v2-react`](/packages/react) with [system props](https://primer.style/components/system-props), making them easier to style in projects that use [styled system](https://styled-system.com/)—like [Primer Components](https://primer.style/components).
+
+## Install
+
+```shell
+npm install @primer/styled-octicons@canary
+```
+
+## Usage
+
+`@primer/styled-octicons`  exports each icons as a named export. This allows you to import only the icons you need:
+
+```jsx
+import {AlertIcon} from '@primer/styled-octicons'
+
+export () => (
+  <AlertIcon color="red.6" mr={2} />
+)
+```
+
+## System props
+
+All icon components in `@primer/styled-octicons` get `COMMON` system props. Read our [System Props](https://primer.style/components/system-props) documentation for a full list of available props.
```

Comment:
Should we explicitly call out support for the `sx` prop here?

Reply:
Yes. Forgot to add that. Nice catch :)